pkg/node/node.go

```diff
@@ -403,7 +404,7 @@ func (n *Node) publishSignature(
 
 			// Check if we failed because someone else submitted in the meantime
 			// or because something wrong happened with our transaction.
```

Comment:
Maybe:
```
// Check if we failed because someone else submitted in the meantime
// or because something wrong happened with our transaction.
// If someone else submitted in the meantime, wait for enough confirmations
// from the chain before making a decision about leaving the submission process.
```
?